packages/graphql/src/types.ts

```diff
@@ -357,6 +357,18 @@ export interface CypherQueryOptions {
     replan?: CypherReplanning;
 }
 
+/** The startup validation checks to run */
+export interface StartupValidationOptions {
+    typeDefs?: boolean;
+    customResolver?: boolean;
```

Comment:
I think this should be `customResolvers` (plural), or maybe even just `resolvers`! The latter is attractive because it mirrors the fields which our users pass into the constructor (`typeDefs` and `resolvers`).

Reply:
I think `resolvers` is the best shout because as you say it mirrors the constructor. I'll update it to that if you're happy!

Reply:
Let's do it!